Library/Formula/c.rb

```diff
@@ -0,0 +1,17 @@
+class C < Formula
+  homepage "https://github.com/ryanmjacobs/c"
+  url "https://github.com/ryanmjacobs/c/archive/v0.07.tar.gz"
+  sha1 "c40044a2747b97a2f8b550c1d417bb03848fcd40"
+
+  depends_on "gnu-sed"
```

Comment:
Okay, I think it works with both GNU and BSD sed now. It for sure works with GNU, but I don't have access to a BSD system.

If you have a BSD system, could you try these commands?
```bash
$ cat > file.c
#!/usr/bin/c blah blah blah
^D
$ sed -i.bak '1!b;s/^#!/\/\/#!/' file.c
$ cat file.c
//#!/usr/bin/c blah blah blah
```